analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -174,7 +175,8 @@ class Analyzer(private val config: AnalyzerConfiguration, private val labels: Ma
             }
         }
 
-        return state.buildResult()
+        val excludes = managedFiles.keys.firstOrNull()?.excludes ?: Excludes()
```

Comment:
Should this re-use `PackageManager.EMPTY_EXCLUDES` (which would need to become at least `internal` then)?

Reply:
As an alternative, would it make sense to have an `EMPTY` constant in the companion object of `Excludes`?

Reply:
Yes, I think that would be more in-line with existing code.

Reply:
Done.